tests/integration_test/chat_service_test.py

```
@@ -820,6 +835,149 @@ def test_chat():
+        json={}
+    )
+    responses.add(
+        "POST", f"https://graph.facebook.com/v2.12/me/messages?access_token={access_token}", json={}
+    )
+    responses.add(
+        "GET", f"https://graph.facebook.com/v2.12/6489091794524304?fields=username&access_token={access_token}",
+        json={"username": "kairon_user_123"}
+    )
+
+    with patch.object(LiveAgentHandler, "check_live_agent_active", _mock_check_live_agent_active):
+        with patch.object(InstagramHandler, "validate_hub_signature", _mock_validate_hub_signature):
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)